pkg/csi/service/wcpguest/controller.go

```diff
@@ -327,6 +346,43 @@ func (c *controller) DeleteVolume(ctx context.Context, req *csi.DeleteVolumeRequ
 			log.Error(msg)
 			return nil, err
 		}
+		// Get the list of persistent volumes from k8s
+		pvList, err := c.pvLister.List(labels.Everything())
```

Comment:
This PR makes CSI controller specific to k8s.

Reply:
Makes sense. I have forwarded the request to bump up external provisioner. Closing this PR